docker/Dockerfile_full

```diff
@@ -94,4 +94,4 @@ COPY sample_configs/config_spacy_duckling.yml ${RASA_NLU_HOME}/config.yml
 EXPOSE 5000
 
 ENTRYPOINT ["./entrypoint.sh"]
-CMD ["start", "-c", "config.yml"]
+CMD ["start", "-c", "config.yml", "--path", "/app/"]
```

Comment:
why are the paths different for the two dockerfiles?

Reply:
shouldn't be. Originally I had it just at `/app/` before realizing before fully realizing what the path command was for. fixing.